src/main/resources/templates/repo/main-repo-shared-resources-template.json.vpt

```diff
@@ -119,6 +140,14 @@
 					}
 				],
 				"StorageEncrypted": true
+                #if((${stack} == 'prod') && (${EnableRdsEnhancedMonitoring} == "true"))
```

Comment:
Maybe we can ignore the "prod" switch here and put this property in the "repo-defaults.properties" that is used to build each DB descriptor. In this way we can set it to false by default and enabled it only in the prod deployment.